ts/components/wallet/TransactionsList.tsx

```diff
@@ -75,7 +80,7 @@ const styles = StyleSheet.create({
 
 export default class TransactionsList extends React.Component<Props> {
   private renderDate(item: Transaction) {
-    const isNew = true; // TODO : handle notification of new transactions @https://www.pivotaltracker.com/story/show/158141219
+    const isNew = this.props.readTransactions[item.id.toString()] === undefined;
```

Comment:
Could you live a comment to explain how isNew is obtained?

Reply:
updated within faf2362d2d0cdb1546a638bb4f9e8a1d5624c7e3